tools/LoadPackage.groovy

```diff
@@ -0,0 +1,252 @@
+            }
+
+            // Expose the workflow/object as a studio template
+
+            // Start by finding the next template dir index
+            def template_dir_name = "1"
+            // If the wkw template dir does not exist, let's create it
+            if (!this.WORKFLOW_TEMPLATES_DIR.exists()) {
+                this.WORKFLOW_TEMPLATES_DIR.mkdirs()
+            }
+            // If it exists, let's iterate over existing templates
+            else {
+                def templates_dirs_list = []
+                new File(this.WORKFLOW_TEMPLATES_DIR_PATH).eachDir { dir ->
+                    templates_dirs_list << dir.getName().toInteger()
+                }
+                if (!templates_dirs_list.isEmpty())
+                    template_dir_name = (templates_dirs_list.sort().last() + 1) + ""
+            }
+            writeToOutput(" Next template dir name " + template_dir_name)
+
+            def studio_template = object.get("studio_template")
+            if (studio_template != null) {
+                // Retrieve the studio template name
+                def studio_template_name = studio_template.get("name")
+
+                // Is the workflow already exposed as a studio template ? -------------
+                def studio_template_found = isAStudioTemplate(studio_template_name, this.WORKFLOW_TEMPLATES_DIR_PATH)
+                writeToOutput(" " + studio_template_name + " found in studio templates ? " + studio_template_found)
+
+                if (!studio_template_found) {
+                    // Create a new template dir in the targeted directory and copy the workflow into it
+                    def template_dir = new File(this.WORKFLOW_TEMPLATES_DIR_PATH, template_dir_name)
+                    template_dir.mkdir()
+                    writeToOutput("] " + template_dir.absolutePath + " created!")
+
+                    // Copy the workflow into it
+                    def file_dest = new File(template_dir, "job.xml")
+                    def file_dest_path = file_dest.absolutePath
+                    Files.copy(Paths.get(object_absolute_path), Paths.get(file_dest_path))
+                    writeToOutput(" " + file_dest_path + " created!")
+
+                    // Create a name file into it
+                    def name_file = new File(template_dir, "name")
+                    name_file.text = studio_template_name
+                    writeToOutput(" " + name_file.absolutePath + " created!")
+
+                    // Create the metadata file into it
+                    def studio_metadata_file = new File(template_dir, "metadata")
+                    studio_metadata_file.text = studio_template.get("offsets_json_string")
+                    writeToOutput(" " + studio_metadata_file.absolutePath + " created!")
+
+                    template_dir_name = (template_dir_name.toInteger() + 1) + ""
+                }
+
+            }
+
+        }
+    }
```

Comment:
The user must be able to load packages independently calling this script... and without necessarily being called from SchedulerStarter (ie without bindings).. Cf Brian comments on the related trello card for the explanations